c/src/ml-api-inference-tizen-privilege-check.c

```diff
@@ -993,6 +993,10 @@ ml_tizen_mm_replace_element (gboolean has_video, gboolean has_audio,
   /* create camcoder handle (primary camera) */
   if (has_video) {
     cam_info.videodev_type = MM_VIDEO_DEVICE_CAMERA0;
+    /**
+     * @note Now network camera is disabled. (cam_info.reserved[0] = 0)
+     * If we need to set net-camera later, discuss with MM team.
+     */
```

Comment:
It seems like the network camera is disabled by ```MMCamPreset cam_info = { 0 };```.
How about moving this comment to line 983?